rust/package/agama.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Aug 21 19:07:28 UTC 2024 - Lubos Kocman <lubos.kocman@suse.com>
+
+- Add product icons. Resolves Issue #1415
```

Comment:
note to all changes file. We use OBS style for github issues ( so OBS can link it properly ). So it should be `gh#openSUSE/agama#1415`